test-suite/coq-makefile/plugin-reach-outside-API-and-fail/run.sh

```diff
@@ -1,36 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
What about the `plugin-reach-outside-API-and-succeed-by-bypassing-the-API` test?

Reply:
This functionality has gone, so I don't know what the test should do.